pages/posts/nepal-2022.md

```diff
@@ -0,0 +1,82 @@
+
+We then headed further up to World Peace Pagoda, it was extremeley serene. The only "noise" was the raindrops sputterning around on the cold marble floor. Part of the charm was it was built at a high altitude lending to the cooler and pleasant climate, and partly because the place itself was quite and well maintained.
+
+<img src="./../../static/img/posts/nepal/peace-pagoda.png">
+
+While riding back to the town, we got caught by the traffic police. The person we rented the scooty from specifically said Indian driving licenses are valid in Nepal and it said something similar on the internet too. But the policemen was adamant and just wouldn't let go. He kept the registration certificate of the vehicle with himself and asked us to tell the owner to collect it himself. Upon reaching the rental place, we got into a huge altercation (verbal) and after paying him for the "damages", we were on our way, on foot back to Baidam Road.
+
+Putting that single bad experience with the locals aside, we went to this place for dinner by the Phewa Lake--everything happens around Phewa Lake in Pokhara--that had good food, a good environment and nive local live music. That was the end of our time in Pokhara, a city I would love to come back to to spend some quality time but by road and not by flight...or so I thought.
+
+The next day as we checked out of the hostel, the adventure sports shop guy called us and asked us whether we want to partake in paragliding. So next thing we know, we were in a cab hailing us from Baidam Road to the top of a hill.
+
+### pokhara -> kathmandu
+The paragliding was a good experience overall but it also meant we were late for the last flight out of Pokhara for the day. We did squander around the airport trying to see if another flight would be scheduled but nope. The next best option was to take a local cab from right outisde the airport to Kathmandu. 
+
+<img src="./../../static/img/posts/nepal/cab-pokhara-accident.png">
+<img src="./../../static/img/posts/nepal/cab-pokhara-inside.png">
+
+So we hopped onto the most normal looking cab and took our seats at the back. It was a 6-seater cab, but the resourceful cabbies loaded upto 10-12 people inside the cab. It was packed to the brim with adults, kids, and lots of luggage. We weren't feeling so good and as luck would have it, a local doctor was also sitting next to us. He slipped us some tablets that he said were for headache. And next thing I personally remember is getting off in Kathmandu at midnight. I was dazed (and later confused) through the whole journey. It was only later that I came to know that the journey was arduous thanks to the terrible roads and that we even had an accident on the way which caused such a delay. Bottomline, there's no good way to travel between Kathmandu and Pokhara, unless of course you're driving on your own.
```

Comment:
I still follow this doctor on instagram, he's in the US now!

Reply:
If it wasn't for him, I don't know how that trip would've went.